src/libs/Notification/LocalNotification/BrowserNotifications.ts

```diff
@@ -34,42 +37,37 @@ function canUseBrowserNotifications(): Promise<boolean> {
 /**
  * Light abstraction around browser push notifications.
  * Checks for permission before determining whether to send.
- * @return resolves with Notification object or undefined
+ *
+ * @param icon Path to icon
+ * @param data extra data to attach to the notification
  */
-function push({title, body, delay = DEFAULT_DELAY, onClick = () => {}, tag = '', icon}: PushParams): Promise<void | Notification> {
-    return new Promise((resolve) => {
-        if (!title || !body) {
-            throw new Error('BrowserNotification must include title and body parameter.');
+function push(title: string, body?: string, icon: string | ImageSourcePropType = '', data: LocalNotificationData = {}, onClick: LocalNotificationClickHandler = () => {}) {
```

Comment:
```suggestion
function push(title: string, body: string, icon: string | ImageSourcePropType = '', data: LocalNotificationData = {}, onClick: LocalNotificationClickHandler = () => {}) {
```

body is required. You can also remove the check inside the function.

Reply:
Leaving `body` optional here since it's not a strict requirement to show a notification. Removed the unnecessary validation though